client/curriculum/html/LinksAndButtons.tsx

```diff
@@ -0,0 +1,882 @@
+          <LessonFigure
+            withBorder
+            src="/images/lessons/links-and-buttons/title-link.png"
+            alt="Atributul title ca link"
+            demo="/demo/html/atributul-title-ca-link"
+          />
+          <p>
+            Acest pop-up nu este atât de accesibil și nu poate fi stilizat. Plus
+            că pe device-urile cu touch este imposibil de folosit. Trebuie să ne
+            asigurăm întotdeauna că oferim un text suficient de descriptiv
+            pentru a ști la ce să ne așteptăm când dăm click pe un link.
+          </p>
+          <blockquote>
+            Putem folosi doar un
+            {' '}
+            <strong className="formatted">icon</strong>
+            {' '}
+            în
+            interiorul unui link?
+          </blockquote>
+          <p>
+            Așa cum am precizat deja, un simplu icon nu ne-ar oferi suficientă
+            informație contextuală despre link, deci nu ar fi chiar recomandat.
+            Dacă totuși suntem nevoiți să folosim un astfel de icon, o bună
+            practică ar fi să utilizam un pattern de genul în care adăugam niște
+            text ascuns ce va oferi context pentru screen readere:
+          </p>
+          <Highlight
+            className="my-5"
+            language="html"
+            code={`
+<a href="/">
+  <!-- Ascundem icon-ul de assistive technology -->
+  <svg viewBox="0 0 32 32" width="32px" height="32px" 
+  aria-hidden="true" focusable="false">
+  <circle cx="16" cy="16" r="16"/>
+  </svg>
+  <!--Se comporta ca un label ce e ascuns -->
+  <span style="position: absolute; opacity: 0">
+  Useful link text
+  </span>
+ </a>`}
+          />
+          <LessonFigure
+            withBorder
+            src="/images/lessons/links-and-buttons/icon-inside-link.png"
+            alt="Doar un icon în interiorul unui link"
+            demo="/demo/html/icon-in-link"
+          />
+          <LessonTip icon={faQuestionCircle}>
+            Atributul
+            {' '}
+            <strong className="formatted">style</strong>
+            {' '}
+            este folosit
+            pentru a adăuga reguli CSS elementelor. Încă nu am ajuns la acel
+            capitol deci e absolut normal să nu știi ce face.
+            <br />
+            {' '}
+            <br />
+            Totuși, te rugăm să-l pui acolo, e un exemplu bun de avut în vedere.
+          </LessonTip>
+          <blockquote>
+            Putem folosi o
+            {' '}
+            <strong className="formatted">imagine</strong>
+            {' '}
+            în
+            interiorul unui link?
+          </blockquote>
+          <p>
+            Daaa. Vom avea nevoie de multe ori să facem asta în proiectele
+            noastre.
+          </p>
+          <Highlight
+            className="my-5"
+            language="html"
+            code={`
+<a href="/">
+  <img src="images/logo.png" alt="Frontend.ro LOGO." />
+</a>`}
+          />
+          <LessonFigure
```

Comment:
Inline demo. We can keep the demo page if we want, but let's make it more "complex" Maybe combine links in pictures with some other features.???